### DISCUSSION TITLE
Marketing

### DISCUSSION THREAD

**Discussion by seekanddefine** [athensresearch/athens#1067] [General]
Dear beautiful humans,

Might you please take a moment to think some marketing kinds of thoughts with me? Don't you adore going to the market doing some markety market marketing? Me too! Let's talk about marketing at Athens. 

I don't know much about marketing, so that means I get to ask questions!

What does marketing at an open-source company look like with options and users groups?

Have you used any open-source marketing automation tools before? 

Are open-source startups any different than other startups?

How do we build a stronger, larger business through appropriate marketing?

Athens solved an immediate and annoying problem for me. What sets us apart from the others out there? Why pick us?

Why will our value proposition resonate with our market? This is an important part of being a strong successful, growing community. Let's pick one and focus until we see it through. 

Who is Athens for? What drives us and how does the data prove it?

How does being open-source add value to customers and inspire confidence in what we are capable of building together?

How do marketing plans change over time in the business world? If we aren't quite agile, how does that affect our available marketing goals? 

What other questions should be here?

I appreciate you, your comments, and your kindness,

-L

**Comment by joelhans** [comment]
I've been thinking about this some 💭.

The obvious most important thing to consider, marketing-wise, is the ideal customer profile and where Athens' value props intersect with that persona. I think that's already happening, although I'm not 100% sure where. It's tricky, because I think there's still a ton of discussion and unknowns about what the product actually will be.

I'm starting to think there's two ways Athens could do some differentiated marketing.

1. Be really open and proud of new releases. I know not every `v1.0.0-beta.xx` release contains some massive new feature or UI overhaul, but other tools in this ecosystem seem . For example, look at how proud [Notion](https://www.notion.so/What-s-New-157765353f2c4705bd45474e5ba8b46c) is of every single little improvement! Even the bugfixes. Or, see the reaction on Twitter to the Reflect founder [adding in search](https://twitter.com/maccaw/status/1390376620729540616). Lots of potential for 🔥 emoji responses if Athens gets a release process going and starts creating marketing material around new/improved features.

2. Provide guides/templates for targeted use cases. Lots of people open up this new brand of note-taking app and immediately get overwhelmed with indecision on how to organize things. They worry that if they set off on the "wrong" path from the beginning, they'll never be able to recover and. Perfection gets in the way of good enough. We could provide sample DBs with some structure/content filled out and a short _How to use this graph_ guide for each. Naturally, [Notion](https://www.notion.so/Notion-Template-Gallery-181e961aeb5c4ee6915307c0dfd5156d#289969a09a894af2a94c94ae4e6fdb15) is a good reference.

I'm pecking at ideas/structure around both of these, which will hopefully congeal in the next few days. Happy to hear feedback, know if the community is interested in one, both, or something else entirely.

**Reply by jefftangx** [reply]
> 1. Be really open and proud of new releases

[Used to be a lot better about this](https://www.notion.so/athensresearch/MVP-Update-Funding-and-Why-I-Started-Athens-e68822f0c3654660ae621cdcbf932bc4#d3031534a8eb4e5a8a96dffe174141cc). Hoping we can restart this with more help :) Could also create a #releases channel like a few other Discords have. So could broadcast updates via email, twitter, discord, and GitHub discussions (there is a button that auto-creates discussions when publishing a release, have only used once I think for first self-hosted RTC PR).

> 2. Provide guides/templates for targeted use cases

Would love templates. Before templates though, we should think of ideal persona, i.e. your 0th point:

> The obvious most important thing to consider, marketing-wise, is the ideal customer profile and where Athens' value props intersect with that persona.

Right now I think our ideal persona is very niche, but this is good for an early product still getting to product-market fit. The narrowest segment might be:

- researcher
- high-tech researcher
- high-tech researchers collaborating
- high-tech researchers collaborating that are privacy-conscious

Our first enterprise customer is exactly this persona. We don't want to dissuade normal notetakers (notetaking is essentially research for a normal person). But this highly segmented user is who would benefit _the most_ from Athens. Not just from the product perspective (networked knowledge, where there are several PKM apps), but for people that really need open-source, self-hosted software with real-time collaboration.

This persona may not have "research" literally in their job title. But they may be at a very early-stage startup (< 10 people, venture-funded). They may have a PhD in a technical field like physics, biology, computer science. Their industry may be technical and regulated: biotech, government agencies (FBI, CIA, DoD), financial services (hedge funds).

Example companies: OpenAI, SpaceX, GoogleX. Alternatively, they may be on the RnD team of a larger company.

Marketing is especially important for our landing page I think. cc @jsmorabito

**Reply by joelhans** [reply]
Put together a rough cut of a release process similar to what I've used in the past. Putting a system like this into operation means there will always be new content for releases, but it definitely "slows" things down a bit. To get it right, there will always be some lag between @tangjeff0 thinking, "I want to push a new release now," and that actually happening.

I imagine most of this discussion happening on the `#releases` channel, unless there's some opposition to using the `#updates` channel for both the auto-generated messages from GitHub _and_ some additional discussion around those messages.

There are two key roles: a **release manager** and a **marketing manager**. These roles don't have to be static; they could be different people with each new release. I titled them _manager_ for a reason, because sometimes these people might be more like shepherds for each process. For example, maybe a community member wants to contribute to a release-related blog post, in which case the marketing manager would prioritize nurturing that contribution over simply doing the work themselves. There may be a need to add additional roles as well.
 
- Pre-release:
    - Release manager notifies `#releases` Discord channel about upcoming release, including rough draft of release notes
    - `#releases` decides on the *headline* feature in the release and whether the release warrants a blog post and/or email
    - Marketing manager edits and expands upon release notes as needed
    - Marketing manager creates visual assets (GIF/MP4 of the feature/improvement in action) to promote new feature(s)
    - Marketing manager writes social posts (Twitter+???)
    - `if blog post == true`, marketing manager writes a blog post including visuals and explanation of/expansion on release notes, schedules post on Ghost
    - `if email == true`, marketing manager drafts an email including the headline feature and visuals
- Release: Release manager publishes release on GitHub, including release notes
- Post-release:
    - notifies `#releases` that new builds are available on GitHub with a link to the release notes
    - `if blog post == true`, marketing manager publishes blog post on Ghost
    - `if email == true`, marketing manager deploys email
    - Marketing manager publishes social
    - Marketing manager updates main `README` on the `athensresearch/athens` repo with new version and release notes
    - Marketing manager updates Handbook with new version and release notes

**Reply by joelhans** [reply]
@tangjeff0 Your insights on the ideal customer has gotten me thinking a lot about branding and messaging. Hopefully I'll have some more time to put some ideas together in the next few days. A much more on-fire issue than templates.

**Reply by jefftangx** [reply]
> To get it right, there will always be some lag between @tangjeff0 thinking, "I want to push a new release now," and that actually happening.

:joy: 

For communication/distribution channels, we have 4 strong channels that we should focus on before expanding to others:

- Twitter (top of funnel), e.g. Twitter update
- GitHub (middle/end of funnel), e.g. Discussion + Changelog
  - We currently have an automated CHANGELOG.md that looks at commit messages. Sometimes I've added screenshots and demos but it was a lot of work. In the past we used [Notion](https://www.notion.so/athensresearch/Updates-and-Release-Notes-DEPRECATED-e18afa006cfd4fec9c462940ac3b84da#fef625622294441380dae99a9d4aba0b). [Canny.io](https://feedback.clickup.com/changelog) has pretty Changelogs. Lots of companies invest in this now like 
- Discord (bottom of funnel, most engaging), e.g. #releases channel
- Email - newest channel but historically best

Talked to @mateicanavra the other day about thinking about the community in terms of funnels and user journeys. Who is the ideal contributor, how do we activate them into getting value from Athens community ASAP, convert them to their first good engagement (e.g. Discord community call or Standup), and retain them. May need some community interviews with ideal Athenians like @limezy @sawhney17 :smile: 

Some resources I (re)-read yesterday that I quite liked!

- https://twitter.com/gregisenberg/status/1393621398090919936 [[Greg Isenberg]]
- https://ethanjb.medium.com/building-a-commercial-open-source-company-part-1-community-market-fit-3069dcac4ebd [[Venrock]] [[Astronomer]]
- https://brianbalfour.com/essays/product-channel-fit-for-growth [[Brian Balfour]]

Make sure to see our [Webflow website](https://athens-research.webflow.io/) to see how we are positioning ourselves!
